shared/ui/Stream/OpenPullRequests.tsx

```diff
@@ -515,6 +518,18 @@ export const OpenPullRequests = React.memo((props: Props) => {
 				// Update default queries for users in a non-destructive way
 				if (derivedState.pullRequestQueries) {
 					patchQueries(defaultQueriesResponse, derivedState.pullRequestQueries, saveQueries);
+					let newObject = {};
```

Comment:
Make it a const and give it a type

```suggestion
					const newObject: FetchProviderDefaultPullResponse = {};
```